pkg/acsengine/defaults.go

```diff
@@ -209,6 +220,24 @@ func setDefaultCerts(a *api.Properties) (bool, error) {
 		a.CertificateProfile.SetCAPrivateKey(caPair.PrivateKeyPem)
 	}
 
+	// Work out the first IP in KubeServiceCidr range
+	_, serviceCidr, err := net.ParseCIDR(a.OrchestratorProfile.KubernetesConfig.ServiceCIDR)
+	if err != nil {
+		return false, err
+	}
+	// Based off https://play.golang.org/p/m8TNTtygK0
+	// Increment the last octet if it's zero.
+	firstIP := func(ip net.IP) net.IP {
+		for j := len(ip) - 1; j >= 0; j-- {
+			if ip[j] == 0 {
+				ip[j]++
```

Comment:
Can you explain the need for this first ip? We didn't have this idea before. Was it covered from the first master ip?